weblate/vcs/git.py

```diff
@@ -654,18 +682,22 @@ def create_pull_request(self, origin_branch, fork_remote, fork_branch):
             head = fork_branch
         else:
             head = "{0}:{1}".format(fork_remote, fork_branch)
-        self.execute(
-            [
-                "pull-request",
-                "--force",
-                "--head",
-                head,
-                "--base",
-                origin_branch,
-                "--message",
-                self.get_merge_message(),
-            ]
+        pr_url = "{}/pulls".format(self.api_url())
+        r = requests.post(
+            pr_url,
+            headers={
+                "Accept": "application/vnd.github.v3+json",
+                "Authorization": "token {}".format(settings.GITHUB_TOKEN),
+            },
+            json={
+                "head": head,
+                "base": origin_branch,
+                "title": self.get_merge_message(),
+            },
         )
+        response = r.json()
+        if not response["url"]:
+            report_error(cause=response["message"])
```

Comment:
What does happen here when the pull request already exists?

Reply:
I haven't tried that. But any time there isn't a success, they usually throw a message (i.e. wrong URL, wrong origin, if repo doesn't exist, etc.) so I am guessing it will give some message here. But I will try and test that out.

Reply:
We want to gracefully handle situation when there is already a pull request opened.